src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -322,7 +331,36 @@ class AppsController(
     }
   }
 
-  private def listRunningTasks(appId: PathId)(implicit identity: Identity): Route = ???
+  private def listRunningTasks(appId: PathId)(implicit identity: Identity): Route = {
+    val maybeApp = groupManager.app(appId)
+    maybeApp.map { app =>
+      authorized(ViewRunSpec, app).apply {
+        val tasksF = instanceTracker.instancesBySpec flatMap { instancesBySpec =>
+          runningTasks(Set(appId), instancesBySpec)
+        }
+        onSuccess(tasksF) { tasks =>
+          complete(Json.obj("tasks" -> tasks.toRaml))
```

Comment:
I thinks we have a Raml object that has `tasks` as a field. Could you create one if we don't have it?